File: backend/src/main/java/com/tripfriend/domain/recruit/recruit/repository/RecruitRepository.java
package com.tripfriend.domain.recruit.recruit.repository;

import com.tripfriend.domain.recruit.recruit.entity.Recruit;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RecruitRepository extends JpaRepository<Recruit, Long>, RecruitRepositoryCustom {
    List<Recruit> findAllByOrderByCreatedAtDesc();
    List<Recruit> findTop3ByOrderByCreatedAtDesc();
}

File: backend/src/main/java/com/tripfriend/domain/recruit/recruit/repository/RecruitRepository.kt
package com.tripfriend.domain.recruit.recruit.repository

import com.tripfriend.domain.recruit.recruit.entity.Recruit
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RecruitRepository : JpaRepository<Recruit, Long>, RecruitRepositoryCustom {
    fun findAllByOrderByCreatedAtDesc(): List<Recruit>
    fun findTop3ByOrderByCreatedAtDesc(): List<Recruit>
}